README.md

```diff
@@ -176,14 +176,14 @@ Publishing a generated terraform provider is possible through the configuration
 
 1. Ensure that the repository you made is called `terraform-provider-{NAME}`, where `NAME` is lowercase. The repository must be public. For this reason, terraform provider generation does not support operating in monorepo mode.
 2. Create and export a signing key to sign your provider releases with. See [Github's documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) for more information. This will need to be generated using either RSA or DSA algorithms. Take note of the following values
-  1. The GPG private key.
-  2. The GPG passphrase.
-  3. The GPG Public Key
-3. Add the ASCII-armored public key to the terraform registry.
-4. Ensure that the following secrets are available to your repository. These will be configured automatically if entered into the speakeasy UI.
-  1. `TERRAFORM_GPG_PRIVATE_KEY`: The GPG private key.
-  2. `TERRAFORM_GPG_PASSPHRASE`: The GPG passphrase.
-5. Once the initial release of your provider is complete (after executing this github action), you will need to manually add the provider to the terraform registry. Follow [the terraform registry instructions](https://registry.terraform.io/publish/provider) to begin this process, and agree to any Terraform Terms and Conditions. You will need to be an organizational admin to complete this step. This step needs only be performed once: subsequent updates will happen automatically.
+3. The GPG private key.
```

Comment:
might want to revert the changes to this list as I think some of these are meant to be indented as per the original